src/Microsoft.Extensions.Logging.EventSource/EventSourceLoggerProvider.cs

```diff
@@ -0,0 +1,241 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.Extensions.Logging.EventSourceLogger
+{
+    /// <summary>
+    /// The provider for the <see cref="EventSourceLogger"/>.
+    /// </summary>
+    internal class EventSourceLoggerProvider : ILoggerProvider
+    {
+        // A small integer that uniquely identifies the LoggerFactory assoicated with this LoggingProvider.
+        // Zero is illegal (it means we are uninitialized), and have to be added to the factory. 
+        private int _factoryID;
+
+        private readonly ILoggerFactory _loggerFactory;
```

Comment:
I think he is talking about the loggerFactory, it is not used anymore. I will make the suggested change.